avaframe/com1DFA/com1DFA.py

```diff
@@ -690,7 +690,7 @@ def initializeSimulation(cfg, demOri, inputSimLines, logName, relThField=''):
         startTimeIni = time.time()
         particles, fields = pI.getIniPosition(cfg, particles, dem, fields, inputSimLines, relThField)
         tIni = time.time() - startTimeIni
-        log.debug('time needed for initialisation %.2f' % tIni)
+        log.info('Ini step for initialising particles finalized, total mass: %.2f, number of particles: %d in %.2f s' % (np.sum(particles['m']), particles['nPart'], tIni))
```

Comment:
Please split this into 2 debug messages:
- info level with the total mass info
- debug level for the rest

Reply:
split into total mass and number of particles for info 
and time for debug - as in the standard initializeParticles